### PR TITLE
Landing 

### PR DESCRIPTION
**Pull request by AndreasInk**
- Landing page
- Apps Section
- Algorithm Section
- About Section
- Mobile Responsive (tested on iPhone 8+ and iPhone 11)

### CODE REVIEW

**Comment by patrickledew** [comment]
Still some minor layout issues, e.g. the Navbar isn't position: fixed so it doesn't scroll with the rest of the content. I have some ideas for the Navbar that I might implement in another branch, so I'm fine merging this as-is.

File: src/components/navbar.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ul {
+    list-style-type: none;
+    margin: inherit;
+    padding: 0;
+    // overflow: hidden;
+    background-color: transparent;
+    margin: 2%;
+}
+
+li {
```
Comment (patrickledew) [inline src/components/navbar.scss:30]: We shouldn't use element selectors without a class attached to them.

File: src/components/navbar.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    display: block;
+    color: white;
+    text-align: left;
+    font-weight: 700;
+    font-size: xx-large;
+
+    text-decoration: none;
+    float: left;
+}
+.smaller {
```
Comment (patrickledew) [inline src/components/navbar.scss:54]: Should really be in `_util.scss`

File: src/components/aboutSection.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  align-self: center;
+  color: black;
+}
+.header {
+  color: black;
+}
+.light {
+  opacity: 0.6;
+}
+sup {
```
Comment (patrickledew) [inline src/components/aboutSection.scss:21]: Avoid raw element selectors

File: src/components/Logo.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  top: 5%;
+  overflow: hidden;
+}
+
+.logo img {
+  width: 100%;
+  height: auto;
+  transition: transform 0.7s ease-out;
+}
+.logo img.beatAnimation:hover {
```
Comment (patrickledew) [inline src/components/Logo.scss:14]: `.beatAnimation` class is redundant and should be removed or renamed

File: src/components/AppsSection.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+import Noti from "../res/Noti.svg";
+import Bolt from "../res/Bolt.svg";
+import Back from "../res/Back.svg";
+
+function AppsSection() {
+  return (
+    <div className="section apps" id="apps">
+      <h1 className="apps-header">Apps</h1>
+
+      <img alt="Apps" className="apps" src={Apps} />
```
Comment (patrickledew) [inline src/components/AppsSection.tsx:12]: Reuse of class `apps`, we already use it for the section div. Maybe change to `apps-img`?

**Review by patrickledew — COMMENTED** [review]
There were a couple minor things I spotted, after we fix these I'm fine with merging.

**Comment by patrickledew** [comment]
Also, a couple linter errors that need to be addressed.

**Comment by patrickledew** [comment]
All patched up. Ready to merge. @AndreasInk

**Comment by AndreasInk** [comment]
Thanks for making those updates, I'll merge

**Pull request merged**